portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,40 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
+  private List<String> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
+    messages.add("Spongebob");
+    messages.add("Gary");
+    messages.add("Mr. Krabs");
+  }
```

Comment:
As of right now, your messages list is static (i.e. never changing) so there's no point really in having this method. Instead you can just set the declaration of the value as...

private static final List<String> messages = Arrays.asList("Spongebob", "Gary", "Mr. Krabs");